pkg/apis/kops/instancegroup.go

```diff
@@ -116,6 +116,10 @@ type InstanceGroupSpec struct {
 	AdditionalUserData []UserData `json:"additionalUserData,omitempty"`
 	// SuspendProcesses disables the listed Scaling Policies
 	SuspendProcesses []string `json:"suspendProcesses,omitempty"`
+	// LoadBalancers to associate with this instance group (AWS ELB)
+	LoadBalancerNames []string `json:"loadBalancerNames,omitempty"`
```

Comment:
Can we create a data-struct for LB and add both Names and Target groups under it, that would also give us the ability to extend that if we ever want to add other LB features without making the IG spec even longer.